src/Installers/Windows/AspNetCoreModule-Setup/CustomAction/aspnetcoreCA.vcxproj

```
@@ -49,7 +49,8 @@
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
   <PropertyGroup>
     <ConfigurationType>DynamicLibrary</ConfigurationType>
-    <PlatformToolset>v142</PlatformToolset>
+    <PlatformToolset Condition="'$(VisualStudioVersion)' == '17.0' ">v143</PlatformToolset>
+    <PlatformToolset Condition="'$(PlatformToolset)' == ''">v142</PlatformToolset>
```

Per "please add a `$(PlatformToolsetVersion)` property in src/Installers/Windows/AspNetCoreModule-Setup/CustomAction/aspnetcoreCA.vcxproj and set `$(PlatformToolset)` unconditionally":

```suggestion
    <PlatformToolsetVersion Condition="'$(VisualStudioVersion)' == '17.0' ">v143</PlatformToolsetVersion>
    <PlatformToolsetVersion Condition="'$(PlatformToolsetVersion)' == ''">v142</PlatformToolsetVersion>
    <PlatformToolset>$(PlatformToolsetVersion)</PlatformToolset>
```

---

So basically move it up into the global category before the props gets imported?